trace_point.rb

```diff
@@ -218,205 +216,202 @@ def self.allow_reentry
+  #     class C; include M; end
+  #     trace = TracePoint.new(:call) do |tp|
+  #       p tp.defined_class #=> M
+  #     end.enable do
+  #       C.new.foo
+  #     end
   #
-  # <b>Note:</b> #defined_class returns singleton class.
+  # <b>Note:</b> #defined_class returns the singleton class.
   #
-  # 6th block parameter of Kernel#set_trace_func passes original class
-  # of attached by singleton class.
+  # The 6th block parameter of Kernel#set_trace_func passes the original class
+  # attached by the singleton class.
   #
   # <b>This is a difference between Kernel#set_trace_func and TracePoint.</b>
   #
-  #	class C; def self.foo; end; end
-  # 	trace = TracePoint.new(:call) do |tp|
-  # 	  p tp.defined_class #=> #<Class:C>
-  # 	end.enable do
-  # 	  C.foo
-  # 	end
+  #     class C; def self.foo; end; end
+  #     trace = TracePoint.new(:call) do |tp|
+  #       p tp.defined_class #=> #<Class:C>
+  #     end.enable do
+  #       C.foo
+  #     end
   def defined_class
     Primitive.tracepoint_attr_defined_class
   end
 
-  # Return the generated binding object from event.
+  # Returns the generated binding object from the event.
   #
-  # Note that for +:c_call+ and +:c_return+ events, the method will return
+  # Note that for +:c_call+ and +:c_return+ events, the method returns
   # +nil+, since C methods themselves do not have bindings.
   def binding
     Primitive.tracepoint_attr_binding
   end
 
-  # Return the trace object during event
+  # Returns the trace object during the event.
   #
-  # Same as the following, except it returns the correct object (the method
+  # Similar to the following, but it returns the correct object (the method
   # receiver) for +:c_call+ and +:c_return+ events:
   #
   #   trace.binding.eval('self')
   def self
     Primitive.tracepoint_attr_self
   end
 
-  #  Return value from +:return+, +:c_return+, and +:b_return+ event
+  # Returns the return value from +:return+, +:c_return+, and +:b_return+ events.
   def return_value
     Primitive.tracepoint_attr_return_value
   end
 
-  # Value from exception raised on the +:raise+ event, or rescued on the +:rescue+ event.
+  # Returns the exception raised on the +:raise+ event or rescued on the +:rescue+ event.
   def raised_exception
     Primitive.tracepoint_attr_raised_exception
   end
 
-  # Compiled source code (String) on *eval methods on the +:script_compiled+ event.
-  # If loaded from a file, it will return nil.
+  # Returns the compiled source code (String) from eval methods on the +:script_compiled+ event.
+  # If loaded from a file, it returns +nil+.
   def eval_script
     Primitive.tracepoint_attr_eval_script
   end
 
-  # Compiled instruction sequence represented by a RubyVM::InstructionSequence instance
+  # Returns the compiled instruction sequence represented by a RubyVM::InstructionSequence instance
   # on the +:script_compiled+ event.
   #
-  # Note that this method is MRI specific.
+  # Note that this method is MRI-specific.
```

Comment:
```suggestion
  # Note that this method is CRuby-specific.
```

This is optional, but I think the current preference is for CRuby instead of MRI.

Reply:
That makes sense, thanks 👍